Assets/HoloToolkit-Examples/ColorPicker/GazeableColorPicker.cs

```diff
@@ -50,5 +36,30 @@ void UpdatePickedColor(PickedColorCallback cb)
             Color col = texture.GetPixel((int)pixelUV.x, (int)pixelUV.y);
             cb.Invoke(col);
         }
+
+        public void OnFocusEnter()
+        {
+            gazing = true;
+        }
+
+        public void OnFocusExit()
+        {
+            gazing = false;
+        }
+
+        public void OnInputUp(InputEventData eventData)
```

Comment:
Sorry I should have thought of this in the interface review but just for handling a simple air tap every class with not have these two stub methods that do nothing. Thoughts?